layouts/default.vue

```diff
@@ -351,6 +368,19 @@ export default {
       this.$store.dispatch('tag/fetchAllTags'),
       this.$store.dispatch('cosmetics/fetchCosmetics', this.$cookies),
     ])
+    if (
+      (this.$auth.user && this.$auth.user.role === 'moderator') ||
+      this.$auth.user.role === 'admin'
```

Comment:
nb: Since I've seen this condition a few times already: A helper method might be useful here. flow could be something like `$auth.user?.isStaff()` or `$auth.user?.canModerate()`